guides/common/modules/proc_configuring-dns-libvirt.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Configuring_dns_libvirt_{context}"]
+= Configuring dns_libvirt
+
+The _dns_libvirt_ DNS provider manages IP reservations and leases using dnsmasq through the libvirt API.
+It uses `ruby-libvirt` gem to connect to the local or remote instance of libvirt daemon.
+
+.Procedure
+. Edit `dns_libvirt.yml`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# Libvirt network; only one network is supported.
+:network: default
+
+# Libvirt connection; ensure the 'foreman-proxy' user has permissions to connect.
+:url: qemu:///system
+----
```

Comment:
Similar to `dhcp_libvirt`, we can use the installer:
```
foreman-installer \
  --foreman-proxy-dns true \
  --foreman-proxy-dns-provider libvirt \
  --foreman-proxy-libvirt-network default \
  --foreman-proxy-libvirt-url qemu:///system
```

And yes, you notice that you can only use a single network + URL if you use both dns_libvirt and dhcp_libvirt.